src/roller/table.ts

```diff
@@ -38,40 +43,148 @@ export class TableRoller extends GenericFileRoller<string> {
+
+        let subTooltips: string[] = [];
+
+        // WARN: we may receive an input that is not string (but a number). Check
+        // for embeded formulas only if we can.
+        if (typeof input?.matchAll === "function") {
+            // Look for dice blocks: `dice: <formula>`
+            const rollerPattern = /(?:\`dice:)(.*?)(?:\`)/g;
+            const foundRollers = input.matchAll(rollerPattern);
+
+            for (let foundRoller of foundRollers) {
+                const formula = foundRoller[1].trim();
+
+                // Create sub roller with formula
+                const subRoller = await this.plugin.getRoller(formula, this.source);
+                // Roll it
+                await subRoller.roll();
+                // Get sub result
+                const rollerResult = await this.getSubResult(subRoller.result);
+
+                // Replace dice block by sub result
+                res.result = res.result.replace(foundRoller[0], rollerResult.result);
+
+                // Update tooltip
+                if (subRoller instanceof TableRoller) {
+                    subTooltips.push(subRoller.combinedTooltip);
+                }
+                else {
+                    const [top, bottom] = subRoller.tooltip.split("\n");
+                    subTooltips.push(top + " --> " + bottom);
+                }
+            }
+        }
+
+        res.combinedTooltip = subTooltips.join(";");
+
+        return res;
+    }
+
     async getResult() {
-        if (this.isLookup) {
-            const result = await this.lookupRoller.roll();
-            const option = this.lookupRanges.find(
-                ([range]) =>
-                    (range[1] === undefined && result === range[0]) ||
-                    (result >= range[0] && range[1] >= result)
-            );
-            if (option) {
-                return option[1];
+        let res = [];
+
+        let subTooltips: string[] = [];
+
+        for (let i = 0; i < this.rolls; i++) {
+            let subTooltip:string;
+            let subResult: SubRollerResult;
+            let selectedOption:string;
+
+            if (this.isLookup) {
+                const result = await this.lookupRoller.roll();
+                const option = this.lookupRanges.find(
+                    ([range]) =>
+                        (range[1] === undefined && result === range[0]) ||
+                        (result >= range[0] && range[1] >= result)
+                );
+                if (option) {
+                    subTooltip = this.lookupRoller.original.trim() + " --> " + `${this.lookupRoller.resultText}${this.header ? " | " + this.header : ""}`.trim();
+                    selectedOption = option[1];
+                }
+            }
+            else {
+                const options = [...this.options];
+                const randomRowNumber = this.getRandomBetween(0, options.length - 1);
+                // TODO: to be confirmed (was this to forbid rolling the same result twice ?)
```

Comment:
I believe that was the case, but I actually don't remember lol

Reply:
OK so I'll remove it; no reason not to be able to get twice the same result for me ;-)